docs/05-technical-reference/00-custom-resources/oper-01-istio.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Istio Manager Operator
+---
+
+The `istios.operator.kyma-project.io` CustomResourceDefinition (CRD) is a detailed description of the kind of data and the format the Istio Manager Operator listens for. To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```shell
+kubectl get crd istios.operator.kyma-project.io -o yaml
+```
+
+More information on the Module Manager you may find [in here](https://github.com/kyma-project/module-manager/).
```

Comment:
how's that information relevant here? what does it bring?

Reply:
Will remove it, not relevant for the customer.